src/Groups/perm.jl

```diff
@@ -89,17 +96,12 @@ julia> g = symmetric_group(4);  s = sylow_subgroup(g, 3)[1];
 julia> number_moved_points(s)
 3
 
-julia> number_moved_points(Int, s)
-3
-
 julia> number_moved_points(gen(s, 1))
 3
 
 ```
 """
-@gapattribute number_moved_points(x::Union{PermGroupElem,PermGroup}) = fmpz(GAP.Globals.NrMovedPoints(x.X))::fmpz
-
-number_moved_points(::Type{T}, x::Union{PermGroupElem,PermGroup}) where T <: IntegerUnion = T(GAP.Globals.NrMovedPoints(x.X))::T
+@gapattribute number_moved_points(x::Union{PermGroupElem,PermGroup}) = fmpz(GAP.Globals.NrMovedPoints(x.X)::GapInt)
```

Comment:
If `fmpz` is called here then the docstring must say `-> fmpz` not `-> Int`.
(The changed test indicates that we do not want to call `fmpz`?)

Reply:
Indeed, good catch! Fixed now